src/emailer.js

```diff
@@ -56,17 +56,35 @@ Emailer.registerApp = function (expressApp) {
 
 	// Enable Gmail transport if enabled in ACP
 	if (parseInt(meta.config['email:smtpTransport:enabled'], 10) === 1) {
-		var smtpOptions = {
-			auth: {
+		var smtpOptions = {};
+
+		if(meta.config['email:smtpTransport:user'] || meta.config['email:smtpTransport:pass']) {
+			smtpOptions.auth = {
 				user: meta.config['email:smtpTransport:user'],
-				pass: meta.config['email:smtpTransport:pass'],
-			},
-		};
+				pass: meta.config['email:smtpTransport:pass']
+			};
+		}
 
-		if (meta.config['email:smtpTransport:serice'] === 'nodebb-custom-smtp') {
+		if (meta.config['email:smtpTransport:service'] === 'nodebb-custom-smtp') {
 			smtpOptions.port = meta.config['email:smtpTransport:port'];
 			smtpOptions.host = meta.config['email:smtpTransport:host'];
-			smtpOptions.secure = true;
+
+			if(meta.config['email:smtpTransport:security'] === 'NONE') {
+				smtpOptions.secure = false;
+				smtpOptions.requireTLS = false;
+				smtpOptions.ignoreTLS = true;
+			}
+			else if(meta.config['email:smtpTransport:security'] === 'STARTTLS') {
```

Comment:
Make sure you run `npm test` before committing, it catches these kinds of linting errors quickly.

Reply:
I thought I had eslint activated in the IDE, but obviously it wasn't. `npm test` sadly doesn't produce anything meaningful on my end: `✖ 70937 problems (70933 errors, 4 warnings)` :smile: 

But I activated eslint now and fixed the errors in emailer.js. Sorry about that, pushing again.